<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convert to EXE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="labelDir">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QRadioButton" name="oneFile">
      <property name="text">
       <string>One File (packs the executable in a single file. Not recommended for debugging.)</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QRadioButton" name="oneDir">
      <property name="text">
       <string>One Directory (packs the executable in a single directory.Distribute files present in the directory 'dist' to clients.)</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="browsebtn_3">
      <property name="text">
       <string>Browse icon</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="browsebtn">
      <property name="text">
       <string>Browse .py</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="browsebtn_4">
      <property name="text">
       <string>Browse python</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QPushButton" name="convert_btn">
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="pyLE">
      <property name="placeholderText">
       <string>Browse for your python file</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="browsebtn_2">
      <property name="text">
       <string>Browse folder</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLineEdit" name="exeLE">
      <property name="placeholderText">
       <string>Choose Python Interpreter to convert with</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="outLE">
      <property name="placeholderText">
       <string>Browse output path</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelPy">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="labelRadio">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="icoLE">
      <property name="placeholderText">
       <string>Browse for icon file</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="labelExe">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
